package.json

```diff
@@ -170,6 +170,7 @@
     "build:dev": "npm run build-demo:dev && npm run build-lib:dev",
     "build-lib:esm": "NODE_ENV=production node scripts/build-esm.mjs",
     "prepublishOnly": "npm run build-lib:prod",
+    "prepare": "npm run build-lib:prod",
```

Comment:
This is the same as `prepublishOnly`

Reply:
it could reference `prepublishOnly`

Reply:
it could, we just need to amke sure to keep that flag `NODE_OPTIONS=--max_old_space_size=4096` up, I remember I had to add it for the heatmap to work well